episodes/ep-1.2-the-research-process.md

```diff
@@ -72,6 +72,9 @@ advising decision-makers, or giving yourself a pat on the back for staying hydra
 
 We’ll cover each of these in detail in later modules of the course, but for now, here’s the big picture:
 
+![An infographic titled "The Research Process" showing 10 stages in a flowchart format. The stages are: 1) Problem Identification – What do I want to know?, 2) Literature Review – What’s already known?, 3) Objectives/Hypothesis – What do I predict?, 4) Research Design – How will I find out?, 5) Data Collection – Go get the facts!, 6) Data Analysis – What do the numbers say?, 7) Result Interpretation – What does it mean?, 8) Conclusion – So what?, 9) Share – Share the story, 10) Evaluation – What worked? What next? Each stage is accompanied by an icon and arranged left to right as a horizontal arrow.](../research-process.png)
```

Comment:
+10 points for accessibility.

Seriously, thank you for taking the time to add this! 🙏